AARA-385:  Buttons extend beyond page header when display resolution is not at maximum

anuroop15 opened the pull request:
#### Which User story does this relate to (link)?
- https://osi-cwds.atlassian.net/browse/AARA-385

#### Brief feature description
-  RFA list view page header buttons to fit within pageheader

#### Does this PR block/effect other stories?
- NA

#### Do any commands needs to be run in order to run the project with this PR?
- NA
 
#### Code coverage percent diff as a result of this PR?
- NA

lauherkcambria commented:
@anuroop15 down to what screen size are we targeting? at 1024x768 the buttons start to blur- what does PO think? 
![image](https://user-images.githubusercontent.com/25331026/42115125-933be558-7ba6-11e8-93dd-72d299a70d32.png)

anuroop15 commented:
@lauherkcambria I have adjusted the css to support 1024 resolution. I initially fixed it for 1200px (laptop) resolutions. Can you re-check the buttons now. I have fixed it.
